### ISSUE TITLE
Control who can view CIS settings menu

### ISSUE THREAD

**Issue by rickhumphries** [elmsln/elmsln#990]
Just wondering how the access permissions for viewing the settings menu is controlled? I'd like to prevent the staff role from being able to see this :) Thanks

![image](https://cloud.githubusercontent.com/assets/4973810/16383304/7ab42038-3c7c-11e6-961c-c46e615cc8db.png)

**Comment by btopro** [comment]
Menu items show up based on role access to each. Well need to throttle access to the views and other things unless we make a specific permission,which we could

Wp8
________________________________

**Comment by rickhumphries** [comment]
Ah yeh - I see now. Managed to reduce them down to all but the 'settings' link. Which permission is it that controls this? It's the only one I can't find

**Comment by btopro** [comment]
Administer CIS settings? Will have to explore a better strategy for using it.

Wp8
________________________________

**Comment by rickhumphries** [comment]
Sure no worries. I used a different user role for the use case I'm after in the end. Couldn't see 'administer CIS settings', I could only find 'Administer ELMSLN settings' which I thought must've been it, so I removed permissions for this from the staff role, but the settings dropdown option still shows.

![image](https://cloud.githubusercontent.com/assets/4973810/16407413/7e50a704-3d0b-11e6-9ce8-fe27846f6c2f.png)
